src/app/Routes.test.tsx

```diff
@@ -202,8 +202,14 @@ describe("Routes", () => {
       route: urls.settings.index,
       state,
     });
-    await waitFor(() =>
-      expect(window.location.pathname).toBe(urls.settings.configuration.general)
+    await waitFor(
+      () =>
+        expect(window.location.pathname).toBe(
+          urls.settings.configuration.general
+        ),
+      {
+        timeout: LONG_TIMEOUT,
```

Comment:
(optional/nitpick) could be worth adding a comment here about this test timing out in coverage and needing the longer timeout